load/resource.go

```diff
@@ -70,11 +70,11 @@ func SetResources(ctx context.Context, g *graph.Graph) (*graph.Graph, error) {
 
 		err := hcl.DecodeObject(&preparer.Source, node.ObjectItem.Val)
 		if err != nil {
+			fmt.Println("error decoding object")
 			return err
 		}
 
 		out.Add(id, preparer)
-
```

Comment:
It's not really on purpose I just have a bad habit of compulsively removing vertical space, I'll try to not do that so much anymore.